package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/central-services-shared",
-  "version": "11.2.0",
+  "version": "11.3.0",
```

Comment:
This needs to match the `package.json` (you can just run an `npm install`)

Reply:
done. I had bumped the minor then reverted to bump the patch instead without syncing.